cms/djangoapps/contentstore/views/item.py

```diff
@@ -1134,6 +1136,7 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
 
     In addition, an optional include_children_predicate argument can be provided to define whether or
     not a particular xblock should have its children included.
+    It can be overridden by setting `OVERRIDE_CREATE_XBLOCK_INFO` to an alternative implementation.
```

Comment:
I don't think it would make sense to have an "alternate implementation" of this, so I suggest re-phrasing this and including an actual example:

```
"""
...
You can customize the behavior of this function using the `OVERRIDE_CREATE_XBLOCK_INFO`
pluggable override point. For example:
    def create_xblock_info(default_fn, xblock, *args, **kwargs):
        xblock_info = default_fn(xblock, *args, **kwargs)
        xblock_info['icon'] = xblock.icon_override
        return xblock_info
"""
```

Reply:
Good idea! Added.